mlflow/pyfunc/scoring_server/client.py

```diff
@@ -79,7 +94,7 @@ def invoke(self, data):
 
 
 class StdinScoringServerClient(BaseScoringServerClient):
-    def __init__(self, process):
+    def __init__(self, process):  # pylint: disable=super-init-not-called
```

Comment:
Why not call super init()?

Reply:
I don't really know 😅  It didn't call super.init() previously and pylint complain in this PR. Maybe I'll just add super.init